de.dashup.shared/src/main/java/de/dashup/shared/Enums/Size.java

```diff
@@ -0,0 +1,37 @@
+package de.dashup.shared.Enums;
+
+public enum Size {
```

Comment:
Why did you remove the size from the widget? I don't remember that something other than a widget can have a size.

Reply:
The widget class way quite big, I wanted to make the code more modular.

Reply:
Maybe a section could have a size one day as well.